enable/testing.py

```diff
@@ -256,7 +256,9 @@ def mouse_move(self, interactor, x, y, window=None,
                                         alt_down=alt_down,
                                         control_down=control_down,
                                         shift_down=shift_down)
-        window.get_pointer_position.return_value = (x, y)
+        if isinstance(window, _MockWindow):
```

Comment:
The `get_pointer_position` function isn't used by most consumers of the `mouse_move` event - in most cases the event position is sufficient.  But you are right, we should check to see if the method is mocked, not the window.